hyperspy/signal.py

```diff
@@ -34,6 +34,7 @@
 from scipy import integrate
 from scipy import signal as sp_signal
 import traits.api as t
+from toolz.itertoolz import concat
```

Comment:
`toolz` is a library originally develop for `dask` and is a requirement of `dask` but it is good practise to add it explicitly as requirement.
It would be better to use `tlz` in order to be able to pick up the C implementation of `cytoolz` when available instead of the pure python in `toolz`.

Reply:
@ericpre that makes sense.  I should have looked at the that  a little closer.  Should I add cytoolz or just toolz to the requirements?